### ISSUE TITLE
Accessibility color contrast issues in style.min.css

### ISSUE THREAD

**Issue by snidersd** [jupyter/jupyter#314]
Also see: [jupyter/notebook#1801](https://github.com/jupyter/notebook/issues/1801)
After running the Dynamic Assessment Plugin for Chrome the following high contrast issues were found:
1. From localhost/tree/docs the contrast between the word docs against the gray background fails. Change the color on line 1094 from 337ab7 to 336db7 or darker passes.
2. Create a new notebook and go back to localhost/tree/docs. The word "running" appears next to the file name of the new notebook, but the color green used for the word "running" fails color contrast. Changing the color on line 9812 from  5cb85c to 5c7f5c or darker passes.
3. From localhost/tree#running (a new notebook is running) the contrast for line 3278 kernel-name   #foad4e needs to be darker.
4. From localhost/tree#running (a new notebook is running) the contrast for line 9814 btn-warning #5bc0de needs to be darker.
5. From a new notebook (http://localhost/notebooks/docs/Untitled.ipynb) the contrast for line 11978 kernel_indicator #777 needs to be darker.

**Comment by snidersd** [comment]
Issue added in error. Closed and moved to Jupyter Notebook GitHub issue#3218
